README.md

```diff
@@ -1,70 +1,88 @@
-# Getting Started with Create React App
+# Project Information
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
-
-## Available Scripts
 
-In the project directory, you can run:
+    This project is frontend part of the STONKS application.
+    Stonks is app that will have capability to predict stock movement based of LR model.
+    Users are able to see current stock parameters.
+    In future crypto may also be supported.
 
-### `yarn start`
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
-
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+## List of Moderators
 
-### `yarn test`
+* **[Mayank Talwar](https://github.com/DreadArceus)**
+* **[Akash Mondal](https://github.com/AkMo3)**: GitHub
+* **[Om Gupta](https://github.com/OmGuptaIND)**: React, Node.js, Django REST, MongoDB, PostgreSQL, SQLite
+* **[Souhardya Malakar](https://github.com/SouhardyaMalakar)**: CSS, React
+* **[Arush Kurundodi](https://github.com/arcinston)**
+* **[Sriraj Behera](https://github.com/SrirajBehera)**: JavaScript, Node, MongoDB
+* **[Utkarsh Choubey](https://github.com/utk2541)**
+* **[Ananjan](https://github.com/ananjan)**
+* **[Shivral](https://github.com/shivral)**: Web Scraping
+* **[Siddhant Singhania](https://github.com/sid11428)**
+* **[Sourabh](https://github.com/sourabhjo7)**
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
-### `yarn build`
+## Members of the LNMIIT-CODING STONKS
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+* [Akash Mondal](https://github.com/AkMo3)
+* [Ananjan](https://github.com/ananjan)
+* [Ansh Pant](https://github.com/AnshPant)
+* [Arush Kurundodi](https://github.com/arcinston)
+* [Astha Singh](https://github.com/Astha-Singh1729)
+* [Ayush Shukla](https://github.com/Ayush-Shukla02)
+* [Mayank Talwar](https://github.com/DreadArceus)
+* [MaySi11](https://github.com/MaySi11)
+* [Pranav Mehta](https://github.com/mehta1351)
+* [N-Coder-Bot](https://github.com/N-coder-bot)
+* [Om Gupta](https://github.com/OmGuptaIND)
+* [Shivral](https://github.com/shivral)
+* [Siddhant Singhania](https://github.com/sid11428)
+* [Souhardya Malakar](https://github.com/SouhardyaMalakar)
+* [Sourabh](https://github.com/sourabhjo7)
+* [Sriraj Behera](https://github.com/SrirajBehera)
+* [Utkarsh Choubey](https://github.com/utk2541)
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+# Setting up Project
```

Comment:
Reduce size to header3 i.e instead of single hash make it `###`

Reply:
Done

Reply:
Not done. Push changes if done locally